src/buffer/buffer.rs

```diff
@@ -457,10 +462,39 @@ mod tests {
     }
 
     #[test]
-    fn debug() {
-        let mut buf = Buffer::empty(Rect::new(0, 0, 12, 2));
-        buf.set_string(0, 0, "Hello World!", Style::default());
-        buf.set_string(
+    fn debug_empty_buffer() {
+        let buffer = Buffer::empty(Rect::ZERO);
+        let result = format!("{buffer:?}");
+        println!("{result}");
+        let expected = "Buffer {
+    area: Rect { x: 0, y: 0, width: 0, height: 0 }
+}";
+        assert_eq!(result, expected);
+    }
+
+    #[cfg(feature = "underline-color")]
```

Comment:
Codecov seems to run without this feature. I mainly used it as it's irrelevant for this change to not write two versions of the same thing.

Reply:
I'm fine with no coverage on the underline-color feature. It's rarely used feature. This might be better:
```suggestion
    #[cfg(not(feature = "underline-color"))]
```

Reply:
It's enabled by default and having this test being active with default features seems smart

Reply:
Got it - it should be enabled in the coverage. Looking at the comment in the makefile and the PR where the feature flag was introduced, it seems this might have just been missed. PR incoming to fix that. No change necessary here.